### ISSUE TITLE
Pretrained Inception V4 to Keras Application Folder 

### ISSUE THREAD

**Issue by gowthamkpr** [keras-team/keras-cv#1153]
Repilca of this [issue](https://github.com/tensorflow/tensorflow/issues/33749) created by @amangalampalli

Would it be possible to add Inception V4 to the Keras Applications folder?

System/User Specifics: Tensorflow 2.0 with tf.keras module

Due to recent discoveries made in neural networks and also according to many benchmarking standards, Inception V4 is one of the best networks available with very high accuracies in both Top 5 and Top 1.

Inception V4 should be included because it has been consistently proven to be one of the most accurate networks for Image Recognition. It will benefit many users who want a network with very high accuracy and also have the resources to run it.

**Comment by shivance** [comment]
@tanzhenyu I would love to take crack at it. I'll refer to [this](https://github.com/tensorflow/models/blob/master/research/slim/nets/inception_v4.py) tensorflow and [this](https://github.com/kentsommer/keras-inceptionV4) keras implementation !

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new KerasHub package, which supports image, text, and multi-modal models. Please read https://github.com/keras-team/keras-hub/issues/1831. KerasHub will support all the core functionality of KerasCV.

KerasHub can be installed with !pip install -U keras-hub. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).

**Comment by innat** [comment]
@sachinprasadhs 
> If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).

Firstly, identifying which feature is required or missing can be done effectively, either by practitioners or the Keras team. Tickets in `keras-cv` were created only when such features were **NOT** available or when a specific need was encountered. Commenting after a significant delay and requesting the recreation of the same issue is neither efficient nor appropriate. Kindly move this ticket (and any other similar tickets with the same context) from `keras-cv` to `keras-hub`.
